core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -677,6 +679,7 @@ object KafkaConfig {
   val ConsumerGroupMaxHeartbeatIntervalMsDoc = "The maximum heartbeat interval for registered consumers."
   val ConsumerGroupMaxSizeDoc = "The maximum number of consumers that a single consumer group can accommodate."
   val ConsumerGroupAssignorsDoc = "The server side assignors as a list of full class names. The first one in the list is considered as the default assignor to be used in the case where the consumer does not specify an assignor."
+  val ConsumerGroupMigrationPolicyDoc = "The config that enables converting the classic group using the consumer embedded protocol to the consumer group using the consumer group protocol and vice versa. " + ConsumerGroupMigrationPolicy.validValuesDescription
```

Comment:
Thanks. I would rather prefer to keep all the documentation defined here. This is what we usually do.